scalalib/src/mill/scalalib/Lib.scala

```diff
@@ -54,20 +41,36 @@ object Lib{
           module = dep.module.copy(
             name =
               dep.module.name +
-              (if (!cross) "" else platformSuffix) +
-              "_" + scalaBinaryVersion(scalaVersion)
+                (if (!cross) "" else platformSuffix) +
+                "_" + scalaBinaryVersion(scalaVersion)
```

Comment:
@ajrnz looks like you're still making unnecessary formatting changes here 😛I don't know what your IDE is doing, but just make sure these are gone by the time your code ends up on a PR. Edit your code in the github web editor if you have to